templates/publisher/market.html

```diff
@@ -33,11 +33,28 @@ <h1 class="u-no-margin--top">{{ title }}</h1>
 
         <div class="row">
           <div class="col-12 u-align--right">
-              <a href="/account/snaps/{{ snap_name }}/market" class="p-button--neutral">Revert</a>
-              <input type="submit" class="p-button--positive" value="Apply" />
+              <input type="submit" class="p-button--neutral" name="submit_revert" value="Revert" />
```

Comment:
We don't really keep any state on client side to revert to, so to cancel changes we need to 'post' but ignore the data and just show flash message that changes were reverted.